services/auth/office/test/helpers/getJwks.test.js

```diff
@@ -0,0 +1,13 @@
+import getJwksUrl from '../../helpers/getjwksUrl';
```

Comment:
This file is identical to `getJwksUrl.test.js` and there is no corresponding `getJwks.js` file, I'm assuming this is left over from a rename and we can remove it.

Reply:
Yes, this is a leftover. Will remove it